### PR TITLE
pgrok: remove installing server-side part

### PR DESCRIPTION
**Pull request by unknwon**
There is no reason to install the server-side part for client installation purpose.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pgrok.rb [hunk 1]
```diff
@@ -25,20 +25,16 @@ def install
       -X main.date=#{time.iso8601}
     ]
 
-    ["pgrokd", "pgrok"].each do |f|
+    ["pgrok"].each do |f|
       system "go", "build", *std_go_args(ldflags: ldflags, output: bin/f), "./cmd/#{f}"
     end
```
Comment (chenrui333) [inline Formula/pgrok.rb:30]: just do this is good enough:
  system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/pgrok"

**Comment by chenrui333** [comment]
> here is no reason to install the server-side part for client installation purpose.

what is the concern of installing `pgrokd`?

**Comment by unknwon** [comment]
> * Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

Not sure how is this supposed to work 🤔 Getting the following error:

```
→ brew audit --strict pgrok
Error: No such file or directory @ realpath_rec - /REDACTED/homebrew-core/pgrok
```

---

> > there is no reason to install the server-side part for client installation purpose.
> 
> what is the concern of installing `pgrokd`?

First of all, it doesn't work and gives the users illusion of "I installed it so it should work". Secondly, exactly as "there is no reason to install the server-side part for client installation purpose", the reason `brew install pgrok` exists is meant for client-side uses.

**Comment by chenrui333** [comment]
> > * Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?
> 
> Not sure how is this supposed to work 🤔 Getting the following error:
> 
> ```
> → brew audit --strict pgrok
> Error: No such file or directory @ realpath_rec - /REDACTED/homebrew-core/pgrok
> ```

can you run `$ brew audit --strict --verbose --debug pgrok`?

works for me though, 

```==> Auditing Formula pgrok on os ventura and arch arm```



> > > there is no reason to install the server-side part for client installation purpose.
> > 
> > 
> > what is the concern of installing `pgrokd`?
> 
> First of all, it doesn't work and gives the users illusion of "I installed it so it should work". Secondly, exactly as "there is no reason to install the server-side part for client installation purpose", the reason `brew install pgrok` exists is meant for client-side uses.

fair enough.

**Comment by chenrui333** [comment]
can you squash the commits into one? I think it is good to. Thanks! 👍

**Comment by unknwon** [comment]
> can you run `$ brew audit --strict --verbose --debug pgrok`?

```
brew audit --strict --verbose --debug pgrok
/opt/homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /opt/homebrew/opt/pgrok/.brew/pgrok.rb
Error: No such file or directory @ realpath_rec - /REDACTED/homebrew-core/pgrok
/opt/homebrew/Library/Homebrew/style.rb:48:in `realpath'
/opt/homebrew/Library/Homebrew/style.rb:48:in `realpath'
/opt/homebrew/Library/Homebrew/style.rb:48:in `block in check_style_impl'
/opt/homebrew/Library/Homebrew/style.rb:48:in `each'
/opt/homebrew/Library/Homebrew/style.rb:48:in `partition'
/opt/homebrew/Library/Homebrew/style.rb:48:in `check_style_impl'
/opt/homebrew/Library/Homebrew/style.rb:35:in `check_style_json'
/opt/homebrew/Library/Homebrew/dev-cmd/audit.rb:185:in `audit'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/call_validation.rb:157:in `call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/call_validation.rb:157:in `validate_call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
/opt/homebrew/Library/Homebrew/brew.rb:94:in `<main>'
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> > can you run `$ brew audit --strict --verbose --debug pgrok`?
> 
> ```
> brew audit --strict --verbose --debug pgrok
> /opt/homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /opt/homebrew/opt/pgrok/.brew/pgrok.rb
> Error: No such file or directory @ realpath_rec - /REDACTED/homebrew-core/pgrok
> /opt/homebrew/Library/Homebrew/style.rb:48:in `realpath'
> /opt/homebrew/Library/Homebrew/style.rb:48:in `realpath'
> /opt/homebrew/Library/Homebrew/style.rb:48:in `block in check_style_impl'
> /opt/homebrew/Library/Homebrew/style.rb:48:in `each'
> /opt/homebrew/Library/Homebrew/style.rb:48:in `partition'
> /opt/homebrew/Library/Homebrew/style.rb:48:in `check_style_impl'
> /opt/homebrew/Library/Homebrew/style.rb:35:in `check_style_json'
> /opt/homebrew/Library/Homebrew/dev-cmd/audit.rb:185:in `audit'
> /opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/call_validation.rb:157:in `call'
> /opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/call_validation.rb:157:in `validate_call'
> /opt/homebrew/Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/sorbet-runtime-0.5.10461/lib/types/private/methods/_methods.rb:270:in `block in _on_method_added'
> /opt/homebrew/Library/Homebrew/brew.rb:94:in `<main>'
> ```

oh, maybe `brew update`. :)

**Comment by unknwon** [comment]
OK, I think I have to run in `/opt/homebrew/opt/` which contains a `pgrok` subdirectory. This command always looking for it under the working directory, maybe something strange with my laptop 🤷‍♂️

**Pull request merged**